MSAL/test/app/ios/MSALTestAppTelemetryDispatcher.m

```diff
@@ -37,4 +37,9 @@ - (void)dispatchEvent:(nonnull NSArray<NSDictionary<NSString *, NSString *> *> *
     }
 }
 
+- (BOOL)onFailureOnly
```

Comment:
Why is this one necessary?

Reply:
Without telemetry supporting this feature, as is right now, unnecessary - Thus, removing it.

Reply:
It isn't!